### DISCUSSION TITLE
2022.09.03 브리핑

### DISCUSSION THREAD

**Discussion by itmakesmesoft** [itmakesmesoft/Steady-Study#60] [Self-Assessment]
이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.
ex)
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점
이유 : (생략)

**Comment by giokim12** [comment]
점수: 40점
쟝고는 또 왜이리 빡셉니까... 민코딩 아직도 week1 못끝낸 나... 
운동은 준영+문삼 오빠랑 복싱 등록했습니다~ 이번주에 2번 갔어요
그리고 지원할 회사들 정리도 하고 지원도 했습니당. 계속 올라오는 중이라 추가할 예정

**Comment by HyeseungNA** [comment]
IM 시험준비 1일 1알고리즘 풀기, Django복습히가
 
점수 : 30점
알고리즘: 4문제 풀었습니다...
Django: 강의 절반 정도 들었습니다...

**Comment by YoungSeok222222** [comment]
주간 목표: IM 시험 대비하기 + django 나가는 동안 밀린 알고리즘 풀기 +
점수: 0점
이유: IM 시험부터해서 알고리즘 풀기까지 어느것 하나 제대로 한 게 없어서 0점을 줬습니다. 더욱 분발하겠습니다.

**Comment by swjiae** [comment]
주간목표 : django 노션 정리하기, 밀린 알고리즘 과제 완료하기, cs공부 틀잡기
점수 : 30점
이유
django 노션 정리하기(TIL에만 대충 정리해놓음) 
밀린 알고리즘 과제 완료하기 X
cs공부 틀잡기 X
주말에 장고 실습도 해야해서 알고리즘 과제 string만 일요일까지 해놓겠습니다.

**Comment by itmakesmesoft** [comment]
이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.

월 목표 : 자바 강의 완강하기
이번 주 목표 : 자바 강의 완강하고 노션 정리하기

점수 : 50점
이유 : 이번 주 즈앵고의 늪에 빠져 자바 강의를 수강할 수 없었음. 오늘 중으로 완강하고 노션에 정리할 계획
월 목표는 절반정도 성공
오늘내로 자바강의 완강하고 9월부터는 자바로 민코딩 기초부터 풀어볼 계획

**Comment by moonthree** [comment]
이번 주 목표 : 브라우저 WEB APIS, 브라우저 DOM, 자바스크립트 이벤트 강의 듣고 블로그에 정리해서 올리기
월 목표 : 브라우저 WEB APIS, 브라우저 DOM, 자바스크립트 이벤트 강의 듣고 블로그에 정리해서 올리기

점수 :  30점
이유 : 쟝고가 할게 많네요; 강의를 듣기만하고 정리해서 올리기는 못했습니다.
일단 듣기는 했으니 30점..

**Comment by sooyeonlee127** [comment]
이번주 목표 : 큐 알고리즘 4 문제 풀기, 블로그 게시글 10개 올리기
월 목표: 과목평가 70점 이상 맞기, IM 테스트 통과를 위해 (https://www.acmicpc.net/workbook/view/1152) 문제 20개 이상 풀기

점수: 40점
이유: 장고 .. 때문에 알고리즘은 손도 대지 못했습니다. 장고 관련 블로그 게시글 5개 올렸습니다. 언제쯤 목표를 달성할 수 있을지 무섭습니다. 월 목표는 절반 정도 달성한 것 같습니다.

**Reply by itmakesmesoft** [reply]
목표를 위해 노력을 했다는 것 만으로도 충분!! 힘내세용!!
